api/krusty/transformerannotation_test.go

```diff
@@ -288,7 +288,7 @@ kind: Pod
 metadata:
   annotations:
     alpha.config.kubernetes.io/transformations: |
-      - repo: https://github.com/kubernetes-sigs/kustomize
+      - repo: https://github.com/kubernetes-sigs/kustomize.git
```

Comment:
For these origin annotations, it looks like the only behavior change introduced by this PR is that we now preserve the `.git` in the repo url. If that's the case, then yes I agree that this is strictly more correct.